src/index.html

```diff
@@ -9,7 +9,63 @@
   <link rel="stylesheet" href="./style.css">
 </head>
 <body>
-  <h1>HTML Form</h1>
+  <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post" >
+    <fieldset class="blocks">
+      <legend>Personal information</legend>
+      <label for="surname">Surname:</label>
+      <input type="text" id="surname" name="surname" class="inputs-fieds" autocomplete="off"><br>
```

Comment:
```suggestion
      <input type="text" id="surname" name="surname" class="inputs-fieds" autocomplete="off">
```
instead of using br, you could add **display: block** property directly to label, and put related input inside it. It gives you possibility to omit **for** and **id** attributes